src/components/organisms/Badges/Badges.tsx

```diff
@@ -104,14 +118,14 @@ export const Badges: React.FC<{
       badges.filter(notEmpty).map((badge) => (
         <li className="badge-list-item" key={badge.label}>
           <Link to={getLocationLink(badge.venue, badge.room)}>
-            <BadgeImage image={badge.image} label={badge.label} />
+            <BadgeImage image={badge.image} />
```

Comment:
Why did we remove the `badge.label` here?

Reply:
It's not being used anymore because of the visual `alt` text flicker.